cli/internal/cmd/manifestVerify.go

```diff
@@ -99,6 +99,11 @@ func cliManifestVerify(cmd *cobra.Command, localSignature string, client getter)
 		return err
 	}
 	remoteSignature := gjson.GetBytes(resp, "ManifestSignature").String()
+
+	if remoteSignature == "" {
+		return errors.New("Coordinator returned no manifest signature. Is the Coordinator in the correct state?")
```

Comment:
shouldn't the coordinator return an error instead?

Reply:
Currently it does not.
Though we may also just add a `RequireState` block to `GetManifest`

Reply:
Do you remember if there is a reason that GetManifestSignature doesn't return the error?
Anyway, your fix is an improvement to the status quo, so I leave it up to you whether to change the Coordinator's behavior or to check for emptiness here.

Reply:
I don't think there ever was a "good" reason, the client API was just never built with the case of no manifest being set when calling this endpoint in mind.
Since changing this would _technically_ break API compatibility, I would keep it as is on the server side for now.